### ISSUE TITLE
RCTFatal + 139 Error on React Native Version 0.17^

### ISSUE THREAD

**Issue by robtg4** [react-native-fellowship/react-native-navbar#126]
RCTFatal + 139 Error on React Native Version 0.17^. Receive the following error: 

![screen shot 2015-12-22 at 8 07 16 am](https://cloud.githubusercontent.com/assets/13613075/11955808/0e6a3ff8-a883-11e5-9e3a-3abce0c9c027.png)

**Comment by grabbou** [comment]
Are you sure module is installed? Because that error happens when it's not in the `node_modules`. I know it's a pretty dumb question, but sometimes it can disappear.

**Comment by adrienbrault** [comment]
I can't install the package:

```
$ npm install react-native-navbar --save
npm WARN peerDependencies The peer dependency react-native@^0.16 included from react-native-navbar will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 15.2.0
npm ERR! argv "/Users/adrienbrault/.nvm/versions/node/v5.3.0/bin/node" "/Users/adrienbrault/.nvm/versions/node/v5.3.0/bin/npm" "install" "react-native-navbar" "--save"
npm ERR! node v5.3.0
npm ERR! npm  v2.14.15
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native@0.8.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-native-blur@0.5.4 wants react-native@>= 0.4 || 0.5.0-rc1 || 0.6.0-rc
npm ERR! peerinvalid Peer react-native-camera@0.3.3 wants react-native@*
npm ERR! peerinvalid Peer react-native-modal@0.3.8 wants react-native@>=0.4.3
npm ERR! peerinvalid Peer react-native-scrollable-tab-view@0.1.10 wants react-native@>=0.8.0 || 0.8.0-rc
npm ERR! peerinvalid Peer react-native-navbar@1.1.2 wants react-native@^0.16

npm ERR! Please include the following file with any support request:
npm ERR!     /Volumes/Developer/react-native/noder-react-native/npm-debug.log
```

My package.json:
```
{
  "name": "x",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "react-native start",
    "test": "mocha --compilers js:babel-core/register --recursive",
    "test:watch": "npm test -- --watch"
  },
  "dependencies": {
    "react": "^0.14.3",
    "react-native": "^0.17.0",
    "react-native-button": "^1.3.1",
    "react-native-lock-ios": "0.0.4",
    "react-native-router-flux": "^1.0.9",
    "react-redux": "^3.1.2",
    "redux": "^3.0.5",
    "redux-logger": "^2.3.1",
    "redux-thunk": "^1.0.2",
    "urijs": "^1.17.0"
  },
  "devDependencies": {
    "babel-core": "^6.3.21",
    "expect": "^1.13.4",
    "mocha": "^2.3.4",
    "react-native-cli": "^0.1.7"
  }
}
```

**Comment by grabbou** [comment]
Yep, that's the issue with `peerDependency` requirement, should be `>= 0.16` instead, cc: @Kureev because of the way `npm` handles pre 1.0 releases, see: https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4
And specifically:
> Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.

**Comment by Kureev** [comment]
Fixed in `1.1.3`
